x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/searchablesnapshots/SearchableSnapshotShardStats.java

```diff
@@ -158,13 +156,13 @@ public CacheIndexInputStats(String fileName, long fileLength, long openCount, lo
             this.cachedBytesRead = cachedBytesRead;
             this.cachedBytesWritten = cachedBytesWritten;
             this.directBytesRead = directBytesRead;
+            this.optimizedBytesRead = optimizedBytesRead;
```

Comment:
This new timed counter is added to track information about optimized read operations executed in DirectBlobContainerIndexInput